libs/components/popovers/testing/src/popover/harness/fixtures/popover-harness-test.module.ts

```diff
@@ -0,0 +1,12 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SkyPopoverModule } from '@skyux/popovers';
+
+import { PopoverHarnessTestComponent } from './popover-harness-test.component';
+
+@NgModule({
+  imports: [CommonModule, NoopAnimationsModule, SkyPopoverModule],
+  declarations: [PopoverHarnessTestComponent],
+})
+export class PopoverHarnessTestModule {}
```

Comment:
I'd remove this module and make `PopoverHarnessTestComponent` standalone, instead.